src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/attribute/form/choices.tsx

```diff
@@ -0,0 +1,41 @@
+import BaseView = require('pimui/js/view/base');
+import React from "react";
+import ReactDOM from "react-dom";
+import Index from 'akeneopimstructure/js/attribute-option/Index';
+
+const __ = require('oro/translator');
+const _ = require('underscore');
+
+class Choices extends BaseView {
+    private config: any;
+
+    initialize(config: any): void {
+        this.config = config.config;
+        BaseView.prototype.initialize.apply(this, arguments);
+    }
+
+    configure(): JQueryPromise<any> {
+        if (_.contains(this.config.activeForTypes, (this.getRoot() as any).getType())) {
```

Comment:
You can use the native method `Array.prototype.includes` instead of the contains function form the Underscore library
```suggestion
        if (this.config.activeForTypes.includes((this.getRoot() as any).getType())) {
```
Furthermore, you can move this test in a dedicated private method like below:
```
isActive(): boolean {
    const attributeType = (this.getRoot() as any).getType() || null;

    if (!attributeType || !Array.isArray(this.config.activeForTypes)) {
       return false;
    }
   
    return this.config.activeForTypes.includes(attributeType);
}
```